migrations/versions/1794e645d265_.py

```diff
@@ -0,0 +1,261 @@
+from sqlalchemy.sql import table, column
+from alembic import op
+from datetime import datetime
+from uuid import uuid4
+from authserver import create_app
+
+
+# revision identifiers, used by Alembic.
+revision = '1794e645d265'
+down_revision = '49bfe630a344'
+branch_labels = None
+depends_on = None
+
+data_trust_table = table('data_trusts',
+                         column('id', sa.String),
+                         column('data_trust_name', sa.String),
+                         column('date_created', sa.TIMESTAMP),
+                         column('date_last_updated', sa.TIMESTAMP))
+
+user_table = table('users',
+                   column('id', sa.String),
+                   column('username', sa.String),
+                   column('firstname', sa.String),
+                   column('lastname', sa.String),
+                   column('organization', sa.String),
+                   column('email_address', sa.String),
+                   column('active', sa.Boolean),
+                   column('data_trust_id', sa.String, sa.ForeignKey('data_trusts.id', ondelete='CASCADE')),
+                   column('password_hash', sa.String),
+                   column('date_created', sa.TIMESTAMP),
+                   column('date_last_updated', sa.TIMESTAMP))
+
+
+oauth2_client_table = table('oauth2_clients',
+                            column('client_id', sa.String),
+                            column('client_secret', sa.String),
+                            column('client_id_issued_at', sa.Integer),
+                            column('client_secret_expires_at', sa.Integer),
+                            column('client_metadata', sa.TEXT),
+                            column('id', sa.String),
+                            column('user_id', sa.String))
+
+
+def hash_password(password: str):
+    return bcrypt.hashpw(password.encode(
+        'utf-8'), bcrypt.gensalt()).decode('utf-8')
+
+
+def upgrade():
+    environment = os.getenv('APP_ENV', None)
+    data_trust_id = str(uuid4()).replace('-', '')
+    admin_user_id = str(uuid4()).replace('-', '')
+
+    if environment == 'PRODUCTION':
+        data_trust_name = os.getenv('DATA_TRUST_NAME', 'New Data Trust')
+        facet_redirect_uri = os.getenv('FACET_REDIRECT_URI', 'http://localhost:8000/auth/redirect')
+        honeybadger_redirect_uri = os.getenv('HONEYBADGER_REDIRECT_URI', 'http://localhost:8000/auth/redirect')
+
+        # Create the new data trust
+        op.bulk_insert(data_trust_table, [
+            {
+                'id': data_trust_id,
+                'data_trust_name': data_trust_name,
+                'date_created': datetime.utcnow(),
+                'date_last_updated': datetime.utcnow()
+            }
+        ])
+
+        # Set up the initial administrative user
+        op.bulk_insert(user_table, [
+            {
+                'id': admin_user_id,
+                'username': 'test_user',
+                'firstname': 'Test',
+                'lastname': 'User',
```

Comment:
Do we really want these "test" credentials for the PRODUCTION environment?

Reply:
Is there a reason why you use ENV variables for the Data Trust Name, but not for the Admin User?

Reply:
Test Credentials for PRODUCTION -> No, that was an oversight. There should be production credentials for production.

ENV for Data Trust Name -> I don't know what the data trust name is, but I do know that I'll always have a "facet_admin" initially. For consistency in deployment.